README.md

```diff
@@ -1,37 +1,60 @@
-4. Ensure you have Python 3.6 or higher by running `python3 -V`. This should return a version number. If your version is less than 3.6, or you get an error, install Python with (MacOS) `brew install python3`. Python is not needed for the core Gatsby system, but is required by several source scripts.
-5. Install yarn and gatsby with `npm i -g gatsby-cli` and `npm i -g yarn`. Gatsby is the software that runs the docs site, and Yarn is an alternative package manager which will replace `npm` for this project.
-6. Install all required packages by simply running `yarn`.
-7. Pull the shared icon files down with `git submodule update --init`
-8. Select sources with `yarn config-sources`, running `yarn pull-sources` as part of this process if prompted (see section below for details)
-9. Run the site locally with `yarn develop`. The site should now be running at `http://localhost:8000/`!
+## MacOS Installation
+We recommend using MacOS to work with the EDB Docs application.
 
-## Running Without a Local Installation (Recommended for Windows)
+1. Install [Homebrew](https://brew.sh/), if it's not already installed. (Use `brew -v` to check.)
 
-If you wish to work with Docs without installing the prerequesites locally, you can do so from within a Docker container using VSCode. See: [Working on Docs in a Docker container using VSCode](README_DOCKER_VSCODE.md)
+1. Install Git using Homebrew with `brew install git`, if it's not already installed.  (Use `git --version` to check.)
+
+1. Clone the [EDB Docs GitHub repo](https://github.com/EnterpriseDB/docs) to your local machine using [GitHub Desktop](https://desktop.github.com/), or via the Terminal with `git clone https://github.com/EnterpriseDB/docs.git`
+
+1. Set up an SSH key in GitHub, if you haven't done so already. (Go to [GitHub's SSH Keys page](https://github.com/settings/keys) to check.) If you don't have an SSH Key set up yet, you'll need to set one up to authenticate you to GitHub. See [GitHub's SSH docs](https://docs.github.com/en/github-ae@latest/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) for more information.
+
+1. Install [Node.js version 14 LTS](https://nodejs.org/en/download/). We recommend using Node version 14 LTS (the Long Term Support release) as version 15 is not compatible with some of our dependencies at this time.
+
+    - If you already have Node installed, you can verify your version by running `node -v` in the cloned repo directory.
+
+    - If you already have Node version 15 installed, you may want to consider using Node Version Manager (NVM) for a simpler way to manage multiple versions of Node.js. Follow the [directions to install NVM](https://github.com/nvm-sh/nvm#installing-and-updating), then run `nvm install` in the cloned repo directory, followed by `nvm use 14` to use version 14 LTS.
```

Comment:
Should this just be "a different version of node" instead of Node 15? Also I think suggesting `nvm use` vs `nvm use 14` makes more sense, because `nvm use` is aware of what version the project needs, vs `nvm use 14` just always loading version 14. Not a big deal, but I'm sure eventually we'll update to 16 lts